cfgov/unprocessed/css/molecules/info-unit.less

```diff
@@ -43,13 +43,13 @@
 .u-info-unit-base__inline( @img_size ) {
   .u-clearfix();
 
-  .m-info-unit_image {
+  .m-info-unit__image {
     float: left;
     margin-bottom: 0;
   }
 
-  .m-info-unit_content,
-  .m-info-unit_heading-text {
+  .m-info-unit__content,
+  .m-info-unit__heading-text {
     margin-left: unit(
       (@img_size + @grid_gutter-width) / @base-font-size-px,
       em
```

Comment:
The print media query on L76 should be changed to `&__image`.

Reply:
Fixed in https://github.com/cfpb/consumerfinance.gov/pull/8274